packages/api-explorer/style/main.scss

```diff
@@ -286,235 +286,6 @@ form.rjsf button[disabled] {
   color: #d2d2d2 !important;
 }
 
-/* Add type labels. */
```

Comment:
None of this junk is used anymore since @rafegoldberg did https://github.com/readmeio/api-explorer/pull/228 🥇